medusa/server/api/v1/core.py

```diff
@@ -281,36 +286,36 @@ def return_help(self):
         except AttributeError:
             self._optionalParams = []
 
-        for paramDict, paramType in [(self._requiredParams, "requiredParameters"),
-                                     (self._optionalParams, "optionalParameters")]:
+        for paramDict, paramType in [(self._requiredParams, 'requiredParameters'),
+                                     (self._optionalParams, 'optionalParameters')]:
 
             if paramType in self._help:
                 for paramName in paramDict:
                     if paramName not in self._help[paramType]:
                         self._help[paramType][paramName] = {}
-                    if paramDict[paramName]["allowedValues"]:
-                        self._help[paramType][paramName]["allowedValues"] = paramDict[paramName]["allowedValues"]
+                    if paramDict[paramName]['allowedValues']:
+                        self._help[paramType][paramName]['allowedValues'] = paramDict[paramName]['allowedValues']
                     else:
-                        self._help[paramType][paramName]["allowedValues"] = "see desc"
-                    self._help[paramType][paramName]["defaultValue"] = paramDict[paramName]["defaultValue"]
-                    self._help[paramType][paramName]["type"] = paramDict[paramName]["type"]
+                        self._help[paramType][paramName]['allowedValues'] = 'see desc'
+                    self._help[paramType][paramName]['defaultValue'] = paramDict[paramName]['defaultValue']
+                    self._help[paramType][paramName]['type'] = paramDict[paramName]['type']
 
             elif paramDict:
                 for paramName in paramDict:
                     self._help[paramType] = {}
                     self._help[paramType][paramName] = paramDict[paramName]
             else:
                 self._help[paramType] = {}
-        msg = "No description available"
-        if "desc" in self._help:
-            msg = self._help["desc"]
+        msg = 'No description available'
+        if 'desc' in self._help:
+            msg = self._help['desc']
         return _responds(RESULT_SUCCESS, self._help, msg)
 
     def return_missing(self):
         if len(self._missing) == 1:
-            msg = "The required parameter: '" + self._missing[0] + "' was not set"
+            msg = 'The required parameter: {0!r} was not set'.format(self._missing[0])
         else:
-            msg = "The required parameters: '" + "','".join(self._missing) + "' where not set"
+            msg = 'The required parameters: {0!r} where not set"'.format("','".join(self._missing))
```

Comment:
I think only the double quote at the end shouldn't be there. I'll do some tests.